docs/user/troubleshooting/plasma.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Plasma Troubleshooting
+summary: Troubleshooting Issues With the Plasma Desktop
+---
+
+# Plasma Desktop Troubleshooting
+
+This contains fixes for various issues you might encounter with the Plasma desktop environment.
+
+## Desktop no longer responds to clicks, cursor stuck in "move" mode (cross)
+
+This is often caused by a Firefox bug. It can be triggered by various things, like moving Firefox windows.
+
+**Symptoms:**
+- Can't click on anything on the desktop.
+- Bringing up windows with keyboard shortcuts (ex: Super for the app menu, F9 for a drop down terminal) works, but they don't get focused. You can't interact with them.
+- Mouse cursor is stuck in move mode, it looks like a cross.
+
+You may not need to reboot to recover from this state. Killing all firefox processes may be all that's necessary. To do so:
+
+1. Open a virtual terminal by pressing `Ctrl+Alt+F3`.
+1. Run the following to kill all firefox windows:
+  ```bash
+  killall firefox
+  ```
+  This should close all Firefox windows.
+1. Return to the Plasma session by pressing `Ctrl-Alt-F2`.
+  You should now be able to use Plasma normally.
+
+## Application windows look corrupt / incomplete
+
+**Symptoms**
+- Application windows look corrupt. For example, NeoChat does not display room or user icons.
+- Menus in an application are blank.
+
+You may be able to fix this by clearing KDE / Plasma cache files, by following the steps below.
+
+:::caution
+
+You should not be logged into your Plasma session while doing this. It is also highly recommended to back up the folders mentioned below before deleting anything.
+
+:::
+
+1. While still logged into your Plasma session, cd to your home directory:
```

Comment:
Don't use commands as verbs. It's confusing for inexperienced users (experienced users won't even need to read the entire step to know what to do)

```suggestion
1. While still logged into your Plasma session, go to your home directory:
```

Reply:
Fixed